### DISCUSSION TITLE
List custom fields in jira issue list?

### DISCUSSION THREAD

**Discussion by kbudde** [ankitpokhrel/jira-cli#687] [Q&A]
Hey,

I want to list epics in a project to retrieve the "Epic Name".
Unfortunately i was not able to find a way to output the epic name with jira issue list command. Is this maybe not supported?

The field is defined in my jira config (auto)
```yaml
epic:
    name: customfield_10005
   ...
```
But listing epic is not showing the field:
`jira issue list -t Epic --columns "summary,name,epic.name,ename,customfield_10005,summary" --plain`

note: adding the customfield to fields.custom (ename) doesn't help as well.

**Comment by kbudde** [comment]
Found related issue: #616

**Reply by ankitpokhrel** [reply]
Hi @kbudde, this is not yet supported unfortunately. I will try to work on this in the near future, PRs are welcome in the mean time. Thanks!

**Reply by QuAzI** [reply]
Is there any way to find all columns supported by `--columns` at the moment? Currently I see only TYPE, KEY, SUMMARY, STATUS and any other fields from `--raw` output are ignored
